webplugin/js/app/components/answer-feedback-service.js

```
@@ -152,22 +140,18 @@ class AnswerFeedback {
         // If valid(0 | 1) feedback is already given then don't show the feedback buttons
         const validFeedback =
             msg.metadata.hasOwnProperty('KM_ANSWER_FEEDBACK') &&
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)